### PR TITLE
Update microstates lists and microstate definition of pKa challenge

### PR DESCRIPTION
**Pull request by MehtapIsik**
Updates for v1.5 of SAMPL6 repository for pKa challenge:

1. Add working definition for pKa microstate: All geometric isomers and resonance structure pairs that have the same total charge and the same number of hydrogens bound to equivalent heavy atoms are related to the same microstate. 
2. Add a comment that only E-isomer of SM20 must be modeled. I did the experiments with pure E-isomer. 
3. Deprecated replicate microstates (mostly related to geometric isomers)
4. Add back `SM21_micro020` microstate which was deprecated by mistake. Bas identified it through Epik predictions.
5. Add lists of suggested physical microstate pairs (microstates pairs that interconvert by one deprotonation event)
6. Adding a canonical SMILES column in addition to canonical isomeric SMILES to `SMXX_microstates.csv` files for easier matching of predicted microstates to microstate IDs. 

@davidlmobley @bannanc Could you please review and merge?

### CODE REVIEW

File: pKa_challenge_instructions.md [hunk 1]
```diff
@@ -133,13 +140,18 @@ It is important that challenge organizers assign unique microstate IDs and keep
 
 ### Updates on microstates lists
 
-Microstate SMILES strings and microstate IDs can be found in [physical_properties/pKa/microstates/](physical_properties/pKa/microstates/) directory. 
-Due to replicate and missing microstates present in the first release of microstate lists, we have updated `SMXX_microstates.csv` files with necessary corrections in Version 1.4 of this repository. 
-The main correction of this update was the removal of resonance structures that were causing dublicate representation of same microstates. 
+Canonical isomeric SMILES, canonical SMILES, and microstate IDs of microstates can be found in [physical_properties/pKa/microstates/](physical_properties/pKa/microstates/) directory. 
+Due to replicate and missing microstates present in the first release of microstate lists, we have updated `SMXX_microstates.csv` files with necessary corrections in Version 1.5 of this repository. 
+
+The main corrections of the update for Version 1.5  was the removal of resonance structures and geometric isomers that were causing dublicate representation of same microstates.
```
Comment (davidlmobley) [inline pKa_challenge_instructions.md]: "dublicate" -> "duplicate"

Comment (davidlmobley) [reply pKa_challenge_instructions.md]: "main corrections" -> "main correction". Also, extra space after "1.5".

Comment (MehtapIsik) [reply pKa_challenge_instructions.md]: Thank you. I corrected these.

**Comment by MehtapIsik** [comment]
OK. I'll wait for her feedback as much as I can today (~4 hours) before merging and sending the announcement email.

**Pull request merged**